packages/cf-component-loading/src/Loading.js

```diff
@@ -3,8 +3,8 @@ const React = require('react');
 class Loading extends React.Component {
   render() {
     return (
-      <div className="cf-loading" role="status">
-        <span className="cf-loading__label">Loading</span>
+      <div className="cf-icon--loading" role="status">
```

Comment:
Modify and use `<Icon>` for this, the classnames are not meant to be part of the public interface. They should be contained solely within the package that creates them.